lib/graphql-bookshelf-bridge/makeResolvers.js

```diff
@@ -15,7 +15,7 @@ export function createResolverForModel (spec, fetcher) {
     transform(attributes, (result, attr) => {
       result[camelCase(attr)] = x => {
         if (typeof x[attr] === 'function') return x[attr]()
-        return x[attr] || x.get(attr)
+        return x[attr] || (x.get && x.get(attr))
```

Comment:
realized there's a bug here predating your change -- will add a fix to this branch